kahuna/public/js/image/view.html

```diff
@@ -84,29 +84,24 @@
                             ng:class="{'image-crop--selected': crop == ctrl.crop}">
                             <a draggable="true"
                                ng:init="extremeAssets = (crop | getExtremeAssets)"
+                               ng-if="ctrl.allowCropSelection(crop)"
```

Comment:
minor: an `ng-switch` may be more obvious here due to the resulting indentation.